### DISCUSSION TITLE
Official Tesla 3rd party program

### DISCUSSION THREAD

**Discussion by itsMeDavidV** [timdorr/tesla-api#734] [General]
A general discussion around the updates and logistics regarding Tesla’s upcoming official 3rd party program.

See tweet for more info: https://twitter.com/teslascope/status/1674158658492702720?s=46&t=wX5DUkfgYCSbQZlGdt712Q

**Comment by Urkman** [comment]
Here is a git repo about this:
https://github.com/teslamotors/fleet-telemetry
